processors/github/test_phase1.sh

```
@@ -0,0 +1,270 @@
+#!/bin/bash
+"""
+Test script for GitHub Processor Phase 1
+
+Tests:
+1. Database initialization with indexes
+2. First run processing (creates chunks)
+3. Idempotency test (detects unchanged)
+4. Deterministic key verification
+5. Vector presence validation
+6. Secret scrubbing verification
+"""
```

Shellcheck (0.10.0)

[error] This is interpreted as a command name containing a linefeed. Double check syntax.

(SC2289)